docs/source/final_attrs.rst

```diff
@@ -0,0 +1,267 @@
+Final names, methods and classes
+================================
```

Comment:
I would call out somewhat early in this section that there's both `Final` (a type qualifier or pseudo-type) and `final` (a decorator).